src/operation/decision_model.jl

```diff
@@ -4,47 +4,46 @@ Default PowerSimulations Operation Problem Type
 struct GenericOpProblem <: DecisionProblem end
 
 """
-    DecisionModel(::Type{M},
-    template::ProblemTemplate,
-    sys::PSY.System,
-    jump_model::Union{Nothing, JuMP.Model}=nothing;
-    kwargs...) where {M<:DecisionProblem,
-                      T<:PM.AbstractPowerFormulation}
+    DecisionModel{M}(
+        template::ProblemTemplate,
+        sys::PSY.System,
+        jump_model::Union{Nothing, JuMP.Model}=nothing;
+        kwargs...) where {M<:DecisionProblem}
 
 This builds the optimization problem of type M with the specific system and template.
 
 # Arguments
 
   - `::Type{M} where M<:DecisionProblem`: The abstract operation model type
-  - `template::ProblemTemplate`: The model reference made up of transmission, devices,
-    branches, and services.
+  - `template::ProblemTemplate`: The model reference made up of transmission, devices, branches, and services.
   - `sys::PSY.System`: the system created using Power Systems
   - `jump_model::Union{Nothing, JuMP.Model}`: Enables passing a custom JuMP model. Use with care
-
-# Output
-
-  - `model::DecisionModel`: The operation model containing the model type, built JuMP model, Power
-    Systems system.
+  - `name = nothing`: name of model, string or symbol; defaults to the type of template converted to a symbol.
+  - `optimizer::Union{Nothing,MOI.OptimizerWithAttributes} = nothing` : The optimizer does
+    not get serialized. Callers should pass whatever they passed to the original problem.
+  - `horizon::Int = UNSET_HORIZON`: Manually specify the length of the forecast Horizon
+  - `warm_start::Bool = true`: True will use the current operation point in the system to initialize variable values. False initializes all variables to zero. Default is true
+  - `system_to_file::Bool = true:`: True to create a copy of the system used in the model.
+  - `initialize_model::Bool = true`: Option to decide to initialize the model or not.
+  - `initialization_file::String = ""`: TODO
+  - `deserialize_initial_conditions::Bool = false`: Option to deserialize conditions
+  - `export_pwl_vars::Bool = false`: True to export all the pwl intermediate variables. It can slow down significantly the solve time.
```

Comment:
```suggestion
  - `export_pwl_vars::Bool = false`: True to export all the pwl intermediate variables. It can slow down significantly the build and solve time.
```